backend/infrahub/database/__init__.py

```diff
@@ -71,7 +78,7 @@ def get_branch_name(branch: Optional[Union[Branch, str]] = None) -> str:
 
 
 class DatabaseSchemaManager:
-    def __init__(self, db: InfrahubDatabase) -> None:
```

Comment:
best to keep all the return statement even if it's None
otherwise ruff and/or mypy will start complaining at some point